service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/NoticeOfChangeEventHandlerEmailTemplateTest.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.fpl.events.NoticeOfChangeEvent;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
+import uk.gov.hmcts.reform.fpl.service.CaseUrlService;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.NoticeOfChangeContentProvider;
+import uk.gov.hmcts.reform.fpl.testingsupport.email.EmailTemplateTest;
+
+import java.util.stream.Stream;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static uk.gov.hmcts.reform.fpl.testingsupport.email.EmailContent.emailContent;
+import static uk.gov.hmcts.reform.fpl.testingsupport.email.SendEmailResponseAssert.assertThat;
+
+@SpringBootTest(classes = {
+    NoticeOfChangeEventHandler.class,
+    NotificationService.class,
+    NoticeOfChangeContentProvider.class,
+    ObjectMapper.class,
+    CaseUrlService.class,
+})
+
+public class NoticeOfChangeEventHandlerEmailTemplateTest extends EmailTemplateTest {
+    private static final Long CASE_ID = 12345L;
+    private static final String CASE_NAME = "Test";
+    private static final String CASE_URL = "http://fake-url/cases/case-details/" + CASE_ID;
+    private static final String SOLICITOR_FIRST_NAME = "John";
+    private static final String SOLICITOR_LAST_NAME = "Watson";
+
+    CaseData caseData = CaseData.builder()
+        .id(CASE_ID)
+        .caseName(CASE_NAME)
+        .build();
+
+    @Autowired
+    private NoticeOfChangeEventHandler underTest;
+
+    @ParameterizedTest
+    @MethodSource("solicitorNameSource")
```

Comment:
tbh dont think you need to have parametrised test here. Different salutation has been verified in unit test so no need to do it here